instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/ClientAddressAndPortExtractor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.http;
+
+import static io.opentelemetry.instrumentation.api.instrumenter.http.ForwardedHeaderParser.extractClientIpFromForwardedForHeader;
+import static io.opentelemetry.instrumentation.api.instrumenter.http.ForwardedHeaderParser.extractClientIpFromForwardedHeader;
+
+import io.opentelemetry.instrumentation.api.instrumenter.network.internal.FallbackAddressPortExtractor;
+
+final class ClientAddressAndPortExtractor<REQUEST>
+    implements FallbackAddressPortExtractor<REQUEST> {
+
+  private final HttpServerAttributesGetter<REQUEST, ?> getter;
+
+  ClientAddressAndPortExtractor(HttpServerAttributesGetter<REQUEST, ?> getter) {
+    this.getter = getter;
+  }
+
+  @Override
+  public void extract(AddressPortSink sink, REQUEST request) {
+    // try Forwarded
+    String forwarded =
+        HttpCommonAttributesExtractor.firstHeaderValue(
```

Comment:
static import to be consistent with the other client class (and improved readability)